src/components/App.tsx

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
+import "./App.css";
 import Termine from './Termine'
 import TerminInput from '../containers/ErfasseTermin'
+import {askForPermissioToReceiveNotifications} from "../push-notification";
 
 const App = () => (
   <div>
     <h1>Termine</h1>
     <Termine />
     <div>Name und Datum eingeben</div>
-    <TerminInput />  
+    <TerminInput />
+      <div className="App">
+          <button onClick={askForPermissioToReceiveNotifications} >
+              Hier klicken, um Push-Notifikation zu testen
+          </button>
+      </div>
```

Comment:
Ich würde dies in die Komponente notification auslagern und hier nur die Komponente einbinden (damit alle Komponenten gleich sind). Dann kann man auch das css, welches jetzt in im app.css ist für die Notifikations-Komponente verwenden in die Komponente verschieben (scheint mir sauberer).

Reply:
Hallo @MichaelEgli vielen Dank für das Review.
Eigentlich ging es mir im ersten Schritt grundsätzlich die Funktionalität zu implementieren. 

Die Integration mit dem bestehenden ServiceWorker hat ziemlich viel Zeit gekostet. 

Aktuell taucht in der Konsole ein MIME type 'text/html' error auf. 

Die Komponente wird natürlich noch angepasst. 
Jedoch bin ich mir nicht sicher, ob es auch als typescript funktioniert.